demo/main.cpp

```diff
@@ -113,7 +113,7 @@ int main(int argc, char* argv[]) {
         for (int i = -round * oversampling; i < (size + round) * oversampling; i++) {
             for (int j = -round * oversampling; j < (size + round) * oversampling; j++) {
                 im(j + round * oversampling, i + round * oversampling) = (std::uint8_t)
-                        pqrs::interpolate_bilinear(gray,
+                        pqrs::interpolate::bilinear(gray,
```

Comment:
I don't think that introducing another namespace would be a good idea; all of the library is "flat" (and that's okay as long as it's small enough). I believe that for the sake of consistency, it would be good not to rename this function.